### PR TITLE
fix the poster image not hiding when video is playing in Chrome

### PR DESCRIPTION
**Pull request by akash1810**
## What does this change?
For some reason, Chrome 57 doesn't like the current css, I think its to do with the background-size property but can't confirm.

As a really hacky fix, lets make the height of the poster 0 to "hide" it when the video is playing/paused.

Removing background-size wasn't an option because the poster image needs to stretch to fill its container (background-size: cover).

Using opacity also wasn't an option as this effects child elements and made the "Watch now" button transparent.

## What is the value of this and can you measure success?
Hosted video pages with a youtube video works again.

## Does this affect other platforms - Amp, Apps, etc?
No.

## Screenshots
Before
before

After
after

### CODE REVIEW

File: static/src/stylesheets/module/commercial/glabs/_hosted-video.scss [hunk 1]
```diff
@@ -134,6 +134,17 @@
...
 }
 
+.youtube__video-buffering,
+.youtube__video-playing,
+.youtube__video-paused {
+    ~ .hosted__youtube-poster-image {
+        // HACK! "hide" the poster image (with a 0 height) when video is playing
+        // but position static for hosted__youtube-play-button's absolute positioning to work
+        height: 0;
+        position: static;
```
Comment (regiskuckaertz) [inline static/src/stylesheets/module/commercial/glabs/_hosted-video.scss]: why not just `display: none`?

**Review by regiskuckaertz — APPROVED** [review]
That's even better than my solution. I'll close my PR 👍

**Review by lps88 — CHANGES_REQUESTED** [review]
Can you take out the `background-size` styles (lines 107 & 151) which was how the poster was initially shown/hidden? And make sure the background size is `cover` always

File: static/src/stylesheets/module/commercial/glabs/_hosted-video.scss (hunk 1 above)
Comment (akash1810) [inline static/src/stylesheets/module/commercial/glabs/_hosted-video.scss]: this element has a child which needs to be shown on video pause, `display: none` effects child elements too

**Comment by akash1810** [comment]
@lps88 good point! I've removed line 151 and changed 107 to be `cover` rather than `0`.

File: static/src/stylesheets/module/commercial/glabs/_hosted-video.scss (hunk 1 above)
Comment (akash1810) [inline static/src/stylesheets/module/commercial/glabs/_hosted-video.scss]: As discussed IRL here's an example of using `display: none` -
 https://github.com/guardian/frontend/compare/aa-fix-yt-poster-display-none?expand=1. It works, though we lose the transition effect 😢 .

**Review by lps88 — APPROVED** [review]
Great 👍

**Pull request merged**